package.json

```diff
@@ -93,7 +93,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint && npm run compile"
+      "pre-commit": "npm run fix && npm run compile"
```

Comment:
I noticed a lot of linting errors. If we fix with every commit, we might reduce the occurrences